### DISCUSSION TITLE
Active Program Sensor after Program is finished

### DISCUSSION THREAD

**Discussion by teddy-rpi** [ekutner/home-connect-hass#183] [Community Support]
Hi there, I am really happy with this integration, thanks a lot for the great work! Currently, my setup consists of a Siemens washing machine and a Siemens dishwasher. Here is my question to you: Lately, I have noticed a strange behavior of the Active Program sensor: From what I recall, the sensor used to turn "unavailable" when the program of the appliance was finished. Now it remains available and still shows the last program as active. Is this a desired behavior? From my opinion, an active program should only be shown when the appliance is currently running, whereas the selected program is shown independent of the status of the appliance.

**Comment by Rep-Hueman** [comment]
This may be related to bug https://github.com/ekutner/home-connect-hass/issues/184 where the values were reversed between active and selected programs. Can you please check if it's working fine in 0.6.0-b2

**Reply by teddy-rpi** [reply]
It seems to be working as expected again, however I didn't update the integration (still on 0.6.0). I just installed HA 2023.3.2 and after that update it was fine. 🤷🏼‍♂️ Thanks anyway!

**Comment by teddy-rpi** [comment]
Hi there, today the washing machine shows an active program again, although it finished 1h ago. The dishwasher is fine and only shows the selected program and no active program. Any suggestions?

**Comment by teddy-rpi** [comment]
> This may be related to bug https://github.com/ekutner/home-connect-hass/issues/184 where the values were reversed between active and selected programs. Can you please check if it's working fine in 0.6.0-b2

How do I install 0.6.0-b2?
